ecs-cluster.yml

```diff
@@ -6,19 +6,20 @@ Parameters:
   EcsInstanceType:
     Type: String
     Description: ECS EC2 instance type
-    Default: t2.micro
+    Default: t2.small
   KeyName:
     Type: String
     Description: Name of existing EC2 KeyPair for SSH to servers
     Default: devops
   VpcId:
-    Type: String
+    Type: AWS::EC2::VPC::Id
     Description: VPC Id of existing VPC
-    AllowedPattern: "^(?:vpc-[0-9a-f]{8}|)$"
+    # QA VpcId: vpc-c116d1bb
     Default: vpc-9c9714f9
   SubnetIds:
-    Type: CommaDelimitedList
-    Description: Comma separated list of existing VPC Subnet Ids
+    Type: List<AWS::EC2::Subnet::Id>
+    Description: List of existing VPC Subnet Ids
+    # QA subnets: "subnet-03540c8937acb1963,subnet-0dc2017d2cfe64e01"
```

Comment:
The subnets in the comments don't match the default value, and we actually have 16 subnets now for 3 EC instances.  

<img width="1166" alt="Screen Shot 2021-01-07 at 9 49 40 AM" src="https://user-images.githubusercontent.com/22908/103906256-aadc1580-50cd-11eb-82ee-d0a29d508abb.png">